test/unittests/libethashseal/EthashTest.cpp

```diff
@@ -48,12 +52,13 @@ BOOST_AUTO_TEST_CASE(calculateDifficultyByzantiumWithoutUncles)
 	header.setNumber(0x2001);
 	header.setTimestamp(130);
 
-	BOOST_REQUIRE_EQUAL(ethash.calculateDifficulty(header, parentHeader), 999024);
+	BOOST_CHECK_EQUAL(ethash.calculateDifficulty(header, parentHeader), 999024);
```

Comment:
Why did you change this?

Reply:
Just pedantic mode. I prefer weaker check if available.